pex/interpreter.py

```diff
@@ -69,16 +69,17 @@ class UnknownRequirement(Error):
 
     @classmethod
     def get(cls, binary=None):
+        # type: (Optional[str]) -> PythonIdentity
+
         # N.B.: We should not need to look past `sys.executable` to learn the current interpreter's
         # executable path, but on OSX there has been a bug where the `sys.executable` reported is
         # _not_ the path of the current interpreter executable:
-        # https://bugs.python.org/issue22490#msg283859
-        if binary and binary != sys.executable:
-            TRACER.log(
```

Comment:
These logs were never viewable since they were emitted durring a subprocess that captures both stdout and stderr as part of the execute_parallel Job / Retain framework. As such, just killed.